DiceKit/Die.swift

```diff
@@ -116,3 +116,14 @@ extension Die: ExpressionType {
 public func == (lhs: Die, rhs: Die) -> Bool {
     return lhs.sides == rhs.sides
 }
+
+// MARK: - Debug
```

Comment:
`CustomDebugStringConvertible`. When we conform to a protocol we state the name as is in the section.